internal/schemautil/schemautil.go

```diff
@@ -119,22 +119,22 @@ func ParseOptionalStringToBool(val interface{}) *bool {
 }
 
 func CreateOnlyDiffSuppressFunc(_, _, _ string, d *schema.ResourceData) bool {
-	return len(d.Id()) > 0
+	return d.IsNewResource()
```

Comment:
Do we still need this? It looks like we can go with ForceNew.

Reply:
> Do we still need this? It looks like we can go with ForceNew.

we want to keep it the way it is in `main` at the moment